thunder/core/transform_common.py

```diff
@@ -848,6 +848,12 @@ def computation(x):
         new_bsyms.append(new_functional_bsym)
         bsym_inplace_to_functional[new_bsym] = new_functional_bsym
 
+        if (
+            len(new_functional_bsym.subsymbols) == 1
+            and new_functional_bsym.rhs == new_functional_bsym.subsymbols[0].rhs
```

Comment:
Could you please explain why name/id/other description not involving explicit construction of `BoundSymbolRHS` is not sufficient? I am biased in favor of using as less of `hash(BoundSymbol...)` as possible...

Reply:
I might be just paranoid but I want to make sure args/kwargs are the same, not only `sym` of them.